### PR TITLE
updating documentation and adding circle build

### PR DESCRIPTION
**Pull request by choldgraf**
This does a couple of things:

1. Split off some of the team compass README into Sphinx documentation (my reasoning here is that website content is more discoverable/inclusive to more people than a github README)
2. Adds the markdown from the last monthly meetings this week
3. Rearranges a bit of content and adds more explanatory text
4. Uses the JupyterHub alabaster theme + builds with circle

### CODE REVIEW

**Comment by choldgraf** [comment]
Built docs: https://6-113493831-gh.circle-artifacts.com/0/html/index.html

**Comment by betatim** [comment]
Moving the content LGTM, let's see if others have an opinion and give them some time to voice it.

File: README.md [hunk 1]
```diff
@@ -12,39 +24,18 @@
...
 ---
 
 ## Planning and operations
 
 Currently, the JupyterHub team is keeping track of the goings-on in the community
-via two main sources: a weekly team report and a monthly team meeting.
+via the following main channels:
 
-*Note: any member of the community is welcome to participate in any of the process around weekly reporting, archiving, and planning team meetings. Input is welcome!*
+* A monthly team meeting
```
Comment (minrk) [inline README.md]: Suggestion!

```suggestion
* A monthly team meeting on the third Thursday of every month
```

File: README.md [hunk 2]
```diff
@@ -57,100 +48,11 @@ are archived in the [weekly reports archive](http://jupyterhub-team-compass.read
...
-### Weekly Team Office Hours
-
-### Monthly videoconference meeting
-
-Feb 8th, 6pm Zurich time
-
-## Team
-
-Min Ragan-Kelley acts as the team lead for the JupyterHub organization.
+**The JupyterHub Team meetings occur on the 3rd Thursday of every month at 6pm Zurich time**. These are open to the community,
```
[the remark this answers is not in the document]
Comment (choldgraf) [reply README.md]: HOLY MOLY THIS CODE SUGGESTION THING IS AMAZING

File: README.md (hunk 1 above)
Comment (choldgraf) [reply README.md]: omg I love this suggestions feature

**Comment by choldgraf** [comment]
all suggestions accepted, without me leaving the github UI. I have chills

**Comment by betatim** [comment]
It is a brave new world!

**Pull request merged**